packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -247,35 +257,74 @@ export class MaterialPanel extends ConnectedLitElement {
     }
   }
 
+  renderVariantsSelector() {
+    return html`
+      <me-section-row
+      label="Variant"
+      style='display: ${
+        getModelViewer().availableVariants.length > 0 ? 'block' : 'none'}'>
```

Comment:
This is fine until you add create variant functionality. Then this length  will change without updating any `@property`, which means it won't cause `render` to run. You  can  cross that bridge when you come to it. Generally I do CSS changes like this outside of the render function to avoid this, since it's cheaper than calling render more often.

Reply:
I see, so you can just update CSS and it will respond without having to 're-render' the html?

Reply:
Yeah gonna leave this here until I have the button, which will provide the hooks to update the CSS display state